tests/ui-toml/float_cmp_change_detection/test.rs

```diff
@@ -0,0 +1,40 @@
+//@no-rustfix
+
+#![deny(clippy::float_cmp)]
+#![allow(clippy::op_ref, clippy::eq_op)]
+
+fn main() {
+    {
+        const C: f64 = 1.0;
+        fn f(x: f64) {
+            let _ = x == C;
+        }
+    }
+    {
+        const fn f(x: f64) -> f64 {
+            todo!()
+        }
+        let _ = f(1.0) == f(2.0);
+    }
+    {
+        let _ = 1.0f32 == 2.0f32;
+        let _ = -1.0f32 == -2.0f32;
+        let _ = 1.0f64 == 2.0f64;
+    }
+    {
+        fn f(x: f32) {
+            let _ = x + 1.0 == x;
```

Comment:
Optional NIT:
It would be nice to have error annotations from `ui_test` here. I assume they are supported for `ui-toml` tests.

Reply:
This PR predates having `//~ lint_name`, but it could be added now.

Reply:
All the test files have now been merged using different revisions instead of different files.